LayoutTests/platform/mac-wk1/imported/w3c/web-platform-tests/mathml/relations/css-styling/writing-mode/writing-mode-002-expected.txt

```diff
@@ -0,0 +1,31 @@
+
+PASS Layout of maction (writing-mode: horizontal-tb; direction: rtl;)
+FAIL Layout of menclose (writing-mode: horizontal-tb; direction: rtl;) assert_approx_equals: inline position (child 0) expected 7.96875 +/- 1 but got 3.1875
```

Comment:
This test seems to assume menclose is treated as an mrow, maybe it should force MathMLCore to be enabled, so we just pass it.